web/gulpfile.js

```diff
@@ -188,7 +188,7 @@ gulp.task('serve', gulp.series('quick', function serve() {
       });
   });
   gulp.watch('app/styles/*.css', gulp.series('css'));
-  gulp.watch('app/scripts/**/*.js', gulp.series('eslint', 'js'));
+  gulp.watch('app/scripts/**/*.js', gulp.parallel('eslint', 'js'));
```

Comment:
Can you explain a bit what's going on in this file and `basic-annotate.js`? I have no idea why you did these changes. Or is it just some API change?

Reply:
> Can you explain a bit what's going on in this file and `basic-annotate.js`? I have no idea why you did these changes. Or is it just some API change?

`gulp serve` was crashing on every change. I guess it was like that since I replaced `ng-annotate` with `basic-annotate`. One mistake was that instead of a new `through.obj` it kept using the same one. I think adding `async` was required due to a Gulp API change.